collect_app/src/androidTest/java/org/odk/collect/android/support/matchers/RecyclerViewMatcher.java

```diff
@@ -69,4 +65,17 @@ public boolean matchesSafely(View view) {
             }
         };
     }
+
+    public static Matcher<View> withListSize(int listId, final int size) {
```

Comment:
This is nice. 

One thing I'm unclear on: Are you intending this to match the total number of items "on screen" or in the `RecyclerView`'s adapter though? If it's the former maybe we should rename to like "withVisibleItems" or something? Otherwise, I believe it needs to call `recyclerView.getAdapter().getItemCount()` instead.

Reply:
You are right I need all elements, fixed.